src/main.py

```
@@ -0,0 +1,293 @@
+        "STARTED": "running",
+        "RETRY": "retry",
+        "FAILURE": "failed",
+        "SUCCESS": "succeeded",
+    }
+    status = state_map.get(result.state, result.state.lower())
+
+    payload = result.result if result.ready() and result.successful() else None
+    finished = None
+    try:
+        finished = result.date_done.isoformat() if getattr(result, "date_done", None) else None
+    except Exception:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)